HospitalToday/Logic/PersonService.cs

```diff
@@ -0,0 +1,31 @@
+﻿using HospitalToday.Domain;
+using HospitalToday.Domain.Person;
+using HospitalToday.Repository;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HospitalToday.Logic
+{
+    class PersonService : IService<Person>
+    {
+        IRepository<Person> personRep;
```

Comment:
It is private by default, but it will be better to make it private read-only. That means, that you will be able to initialize variable in constructor, but in other places it will be read-only.

Reply:
Fixed, I added the key word - readonly.